index.js

```diff
@@ -1,56 +1,52 @@
-// Requires
-var util = require('util');
-var EventEmitter = require('events').EventEmitter;
-
-// Constructor function
-function PIR (hardware, callback) {
-  var self = this;
-
-  // Set properties
-  self.hardware = hardware; // Hardware should be a specific pin for PIR
-  self.movement = false; // Initialize as not moving
-
-  // Begin listening for events
-  self.hardware.on('rise', function (time) {
-    self.emit('movement', time);
-    self.movement = true;
-  });
-
-  self.hardware.on('fall', function (time) {
-    self.emit('stillness', time);
-    self.movement = false;
-  });
-
-  self.hardware.on('change', function (time, type) {
-    self.emit('change', time, type);
-  });
-
-  // Emit the ready event
-  setImmediate(function emitReady() {
-    self.emit('ready', self);
-    if(callback) {
-      callback(null, self);
-    }
-  });
-}
-
-// Inherit event emission
-util.inherits(PIR, EventEmitter);
+'use strict';
+
+const Emitter = require('events').EventEmitter;
+
+class PIR extends Emitter {
+  constructor(hardware, callback) {
+    super();
+
+    // Hardware should be a specific pin for PIR
+    this.hardware = hardware;
+    // Initialize as not moving
+    this.movement = false;
+
+    // Begin listening for events
+    this.hardware.on('rise', () => {
+      const now = Date.now();
+      this.movement = true;
+      this.emit('movement', now);
+      this.emit('movement:start', now);
+    });
+
+    this.hardware.on('fall', () => {
+      const now = Date.now();
+      this.movement = false;
+      this.emit('stillness', now);
+      this.emit('movement:end', now);
+    });
+
+    this.hardware.on('change', state => {
+      const now = Date.now();
+      this.movement = !!state;
+      this.emit('change', now, state);
+    });
+
+    setImmediate(() => {
+      this.emit('ready', this);
+      if (callback) {
+        callback(null, this);
+      }
+    });
+  }
 
-// Functions
-// Read the state of the pin
-PIR.prototype.read = function (callback) {
-  if(callback) {
-    callback(this.hardware.read());
+  static get PIR() {
+    return PIR;
   }
-  return this.hardware.read();
```

Comment:
Synchronous reads aren't supportable on Tessel 2, and this isn't really useful for the sensor itself (the `movement` property and related events should be sufficient for all use cases)